releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -442,7 +442,8 @@ private static List<ReleaseNotesMessage> createAllNotes() throws Exception {
             createReleaseNotesMessage("Mock issue title 7 thisIssueTitleIsExactly87Characters"
                 + "LongAndThenYouThe13ChrIndentation", "Author 10"),
             createReleaseNotesMessage("Mock issue title 8 thisIssueTitleIsExactly100Characters"
-                + "LongAndWeExpectItToGetWrappedDueToBeingTooLng", "Author 11"));
+                + "LongAndWeExpectItToGetWrappedDueToBeingTooLng", "Author 11"),
+            createReleaseNotesMessage("Mock issue title 9 escape @ and @@@@@", "Author 12"));
```

Comment:
Please point to or add example tests that contain `<`, `>`, `&` in the release notes.

Reply:
Those should be escaped as well then, right?

Reply:
Done.